enterprise/models.py

```diff
@@ -67,7 +67,7 @@ class Meta(object):
     name = models.CharField(max_length=255, blank=False, null=False, help_text=_("Enterprise Customer name."))
     active = models.BooleanField(default=True)
     history = HistoricalRecords()
-
+    identity_provider = models.SlugField(null=True, blank=True, default=None)
```

Comment:
@asadiqbal08 We discussed this on another PR: https://github.com/edx/edx-enterprise/pull/11#discussion_r88712167 I think that this should have `unique=True`, though it requires a bit of work to make that possible.

I think we don't want to allow multiple IdPs per EC; we want to force that to be 1:1. (If we ever needed to allow multiple ECs for one IdP we could just create two IdPs with identical values but different IDs. I don't think we'll need that though.)